lib/unleash_proxy_client_flutter.dart

```diff
@@ -136,8 +142,11 @@ class UnleashClient extends EventEmitter {
       this.refreshInterval = 30,
       this.fetcher = get,
       this.poster = post,
+      // should be used only for testing
       this.sessionIdGenerator = generateSessionId,
-      this.eventIdGenerator = generateEventId,
+      // should be used only for testing
+      this.idGenerator = generateId,
+      // should be used only for testing
```

Comment:
What's with the comments here? I'm not sure I understand what you mean?

Reply:
some constructor parameters are exposed only to make testing easier but should not be used in production code